tests/flow/test_index_updates.py

```diff
@@ -176,3 +176,21 @@ def test06_remove_indexed_prop(self):
         expected_result = []
         self.env.assertEquals(result.result_set, expected_result)
 
+    def test07_update_exact_and_fulltext_indexes(self):
+        # Remove the exact-match index on a property
+        redis_graph.redis_con.execute_command("GRAPH.QUERY", GRAPH_ID, "DROP INDEX ON :label_a(group)")
```

Comment:
Why is the exact-match index being removed?
Aren't we testing that both exact-match and full-text are updated?

Should we incorporate original issue scenario?

Reply:
If we didn't remove the exact-match index, both indexes would be updated properly on the current master.

The exact error case is:
1) The label has exact-match and full-text indexes.
2) A property is indexed under the full-text index but not the exact-match index.
3) `Schema_GetIndex` only checks the exact-match index, and since the property is not indexed under it, returns NULL (leading `OpUpdate` to believe that it doesn't need to perform reindexing).